src/Compilers/Core/Portable/CodeGen/ReferenceDependencyWalker.cs

```diff
@@ -145,6 +145,25 @@ private static void VisitMethodReference(Cci.IMethodReference methodReference, E
             }
         }
 
+        internal static void VisitSignature(Cci.ISignature signature, EmitContext context)
```

Comment:
>VisitSignature [](start = 29, length = 14)

Consider adjusting the previous method to share the code with this one, by calling this method for the same set of operations.

Reply:
> Consider adjusting the previous method to share the code with this one, by calling this method for the same set of operations.

The reason I didn't do that is because that intersperses the ExtraArguments check, which `ISignature` does not have.

---
In reply to: [382332630](https://github.com/dotnet/roslyn/pull/41794#discussion_r382332630) [](ancestors = 382332630)